docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -0,0 +1,155 @@
+In this second, we show you how to create a new local Miden account and how to receive funds from the public Miden faucet website.
+
+## Configure the Miden client
+
+The Miden client facilitates interaction with the Miden rollup and provides a way to execute and prove transactions. 
+
+!!! tip
+      Check the [Miden client documentation](https://docs.polygon.technology/miden/miden-client/cli-reference/) for more information.
+
+1. Clone the Miden client.
+
+      ```shell
```

Comment:
I think we should not ask the user to clone the repo anymore. 

We should do it as simple as described in here 

https://docs.polygon.technology/miden/miden-client/install-and-run/

Reply:
OK, but where do they find the toml file in this case? @Dominik1999

Reply:
@igamigo how do you change the Node in case you install the client?

Reply:
OK, then we keep the tutorial as is for now, that means we keep the cloning and installing part. 

And we can change it if we can skip the cloning at some point